src/classes/club.py

```diff
@@ -13,7 +19,28 @@ def __str__(self):
         return f"Club with chat_id: {self.chat_id}"
 
     def data_to_json(self):
-        data = {"chat_id": self.chat_id, "title": self.title}
-        return data
+        return {
+            "chat_id": self.chat_id,
+            "title": self.title,
+            "teams": [team.data_to_json() for team in self.teams],
+        }
+
+    def add_team(self, team):
+        self.teams.append(team)
+
+    def remove_team(self, team_id: int):
+        team_to_remove = next(
+            (team for team in self.teams if team.team_id == team_id), None
```

Comment:
Let's either pass the removing team as a parameter or combine this method with "has_team" because we're performing the same search in "teams" twice